spk/homeassistant/src/dsm-control.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python3"
+PYTHONPATH="${INSTALL_DIR}/share/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="homeassistant"
+PYTHON="${INSTALL_DIR}/env/bin/python3"
+
+CONFIG_DIR="${INSTALL_DIR}/etc"
```

Comment:
Imo, let's stick to the approach we use for most packages. ${INSTALL_DIR}/var is the home directory for the user you created (see installer.sh).
Its normally used for config files, (sqlite) databases, log files and all that kind of stuff. There's no harm in putting a PID file in there.
Depending on how the log files are generated, sometimes it makes sense to create a ${INSTALL_DIR}/var/log directory (e.g. when those are rotated)

As for PYTHONPATH, that is usually used to specify the path to the Python executable (not to the executable you want to run). The reason I mentioned it is because you're using PYTHONPATH where others would use PATH. 
Besides that, you're calling (or should be calling) `${PYTHON} ${HOMEASSISTANT}`. It's quite likely you don't even need to specify PYTHONPATH or PATH.